compiler/rustc_smir/src/rustc_internal/internal.rs

```diff
@@ -297,6 +470,7 @@ where
         (*self).internal(tables)
     }
 }
+
```

Comment:
nit

Reply:
Do you want me to remove this? I added a space here since these two implementations had no space in between.

Reply:
ah no problem then